File: backend/MeetHut.Services/Application/DTOs/UserDTO.cs
using System;

namespace MeetHut.Services.Application.DTOs
{
    /// <summary>
    /// User DTO object
    /// </summary>
    public class UserDTO
    {
        /// <summary>
        /// Id
        /// </summary>
        public int Id { get; set; }
        
        /// <summary>
        /// Unique user name
        /// </summary>
        public string UserName { get; set; }
        
        /// <summary>
        /// Email address
        /// </summary>
        public string Email { get; set; }
        
        /// <summary>
        /// Full name
        /// </summary>
        public string? FullName { get; set; }

        /// <summary>
        /// Last login date
        /// </summary>
        public DateTime LastLogin { get; set; }
    }
}

File: MeetHut.Backend/ClientApp/src/app/web/dtos/user.dto.ts
export class UserDTO {
  id: number;
  userName: string;
  email: string;
  fullName: string;
  lastLogin: Date;

  constructor(
    id: number,
    userName: string,
    email: string,
    fullName: string,
    lastLogin: Date
  ) {
    this.id = id;
    this.userName = userName;
    this.email = email;
    this.fullName = fullName;
    this.lastLogin = lastLogin;
  }
}
